lib/Utils/Architectures.pm

```diff
@@ -42,21 +48,73 @@ our %EXPORT_TAGS = (
 
 # specific architectures
 
+=head2 
```

Comment:
All head2 lines must contain the function name (without brackets or parameters).

eg. here:
```
=head2 is_s390x
```